data/clearance_astrazeneca/meta.yaml

```diff
@@ -0,0 +1,66 @@
+---
+name: clearance_astrazeneca
+description: |-
+    Drug clearance is defined as the volume of plasma cleared of a drug
+    over a specified time period and it measures the rate at which the active drug
+    is removed from the body. This is a dataset curated from ChEMBL database containing
+    experimental results on intrinsic clearance, deposited from AstraZeneca. It
+    contains clearance measures from two experiments types, hepatocyte and microsomes.
+targets:
+    - id: drug_clearance
+      description: the volume of plasma cleared of a drug over a specified time period
+      units: mL / (min g)
+      type: continuous
+      names:
+          - noun: drug clearance
+          - noun: volume of plasma cleared of a drug over a specified time period
+      uris:
+          - http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C94618
+benchmarks:
+    - name: TDC
+      link: https://tdcommons.ai/
+      split_column: split
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: chembl_id
+      type: Other
+      names:
+          - ChEMBL id
+          - ChEMBL identifier number
+      description: ChEMBL ids
```

Comment:
Also here I wonder about the ID. Do we need an attribute that prevents sampling this column?

Reply:
If we always use `chembl_id` as col id we could simply exclude in our sampling logic?

Reply:
or we simply comment out those lines so we have it should we need them again?

Reply:
added separate flag